src/cryptoadvance/specter/helpers.py

```diff
@@ -241,16 +241,38 @@ def generate_mnemonic(strength=256):
     return words
 
 
+def wallet_type_by_slip132_xpub(xpub):
+    """
+    see: https://github.com/satoshilabs/slips/blob/master/slip-0132.md
+    Electrum backups use SLIP-132 but note that other wallets don't make the same
+    guarantee.
+    """
+    if xpub.startswith("xpub") or xpub.startswith("tpub"):
+        return "sh"
```

Comment:
It can be either `sh` or `pkh` depending if it's multisig or single-sig.
We don't support it, but I think this should work for that as well. Maybe we can return `legacy` and then handle it accordingly depending on the number of keys we detected below?

Reply:
@stepansnigirev Now updated with an `is_multisig` var to pass into that method.